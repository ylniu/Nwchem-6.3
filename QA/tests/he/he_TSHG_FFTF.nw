echo
start he_dat

memory heap 200 mb

geometry
  he 0 0 0
end

basis
 * library cc-pVDZ
end

task scf energy
