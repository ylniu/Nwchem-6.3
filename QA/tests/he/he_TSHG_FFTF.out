All connections between all procs tested: SUCCESS
 argument  1 = he_TSHG_FFTF.nw



============================== echo of input deck ==============================
echo
start he_dat

memory heap 200 mb

geometry
  he 0 0 0
end

basis
 * library cc-pVDZ
end

task scf energy
================================================================================


                                         
                                         


              Northwest Computational Chemistry Package (NWChem) 6.1
              ------------------------------------------------------


                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352

                              Copyright (c) 1994-2010
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute

             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file

                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname      = arcen
    program       = ../../../bin/LINUX64/nwchem
    date          = Tue Feb 14 15:40:20 2012

    compiled      = Tue_Feb_14_15:40:14_2012
    source        = /home/d3y133/nwchem-dev/nwchem-trunk
    nwchem branch = Development
    input         = he_TSHG_FFTF.nw
    prefix        = he_dat.
    data base     = ./he_dat.db
    status        = startup
    nproc         =        1
    time left     =     -1s



           Memory information
           ------------------

    heap     =   26214401 doubles =    200.0 Mbytes
    stack    =   13107201 doubles =    100.0 Mbytes
    global   =   26214400 doubles =    200.0 Mbytes (distinct from heap & stack)
    total    =   65536002 doubles =    500.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------

  0 permanent = .
  0 scratch   = .




                                NWChem Input Module
                                -------------------



 Scaling coordinates for geometry "geometry" by  1.889725989
 (inverse scale =  0.529177249)



                             Geometry "geometry" -> ""
                             -------------------------

 Output coordinates in angstroms (scale by  1.889725989 to convert to a.u.)

  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 he                   2.0000     0.00000000     0.00000000     0.00000000

      Atomic Mass 
      ----------- 

      he                 4.002600


 Effective nuclear repulsion energy (a.u.)       0.0000000000

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000     0.0000000000     0.0000000000


            XYZ format geometry
            -------------------
     1
 geometry
 he                    0.00000000     0.00000000     0.00000000



 Summary of "ao basis" -> "" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 *                          cc-pVDZ                   on all atoms 


                      Basis "ao basis" -> "ao basis" (cartesian)
                      -----
  he (Helium)
  -----------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  3.83600000E+01  0.023809
  1 S  5.77000000E+00  0.154891
  1 S  1.24000000E+00  0.469987

  2 S  2.97600000E-01  1.000000

  3 P  1.27500000E+00  1.000000



 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 he                         cc-pVDZ                  3        5   2s1p


                                 NWChem SCF Module
                                 -----------------



  ao basis        = "ao basis"
  functions       =     5
  atoms           =     1
  closed shells   =     1
  open shells     =     0
  charge          =   0.00
  wavefunction    = RHF 
  input vectors   = atomic
  output vectors  = ./he_dat.movecs
  use symmetry    = F
  symmetry adapt  = F


 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 he                         cc-pVDZ                  3        5   2s1p



 Forming initial guess at       0.0s


      Superposition of Atomic Density Guess
      -------------------------------------

 Sum of atomic energies:          -2.85516045

      Non-variational initial energy
      ------------------------------

 Total energy =      -2.855160
 1-e energy   =      -3.882025
 2-e energy   =       1.026865
 HOMO         =      -0.914148
 LUMO         =       1.397442


 Starting SCF solution at       0.0s



 ----------------------------------------------
         Quadratically convergent ROHF

 Convergence threshold     :          1.000E-04
 Maximum no. of iterations :           30
 Final Fock-matrix accuracy:          1.000E-07
 ----------------------------------------------


 #quartets = 2.100D+01 #integrals = 4.500D+01 #direct =  0.0% #cached =100.0%


 Integral file          = ./he_dat.aoints.0
 Record size in doubles =  65536        No. of integs per rec  =  43688
 Max. records in memory =      2        Max. records in file   =  12281
 No. of bits per label  =      8        No. of bits per value  =     64


              iter       energy          gnorm     gmax       time
             ----- ------------------- --------- --------- --------
                 1       -2.8551604773  4.95D-08  4.95D-08      0.0


       Final RHF  results 
       ------------------ 

         Total SCF energy =     -2.855160477254
      One-electron energy =     -3.882025095285
      Two-electron energy =      1.026864618031
 Nuclear repulsion energy =      0.000000000000

        Time for solution =      0.0s


             Final eigenvalues
             -----------------

              1      
    1   -0.9141
    2    1.3974
    3    2.5244
    4    2.5244
    5    2.5244

                       ROHF Final Molecular Orbital Analysis
                       -------------------------------------

 Vector    1  Occ=2.000000D+00  E=-9.141479D-01
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 3.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1      0.592616  1 He s                  2      0.513027  1 He s          

 Vector    2  Occ=0.000000D+00  E= 1.397442D+00
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 7.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      1.187231  1 He s                  1     -1.149573  1 He s          

 Vector    3  Occ=0.000000D+00  E= 2.524372D+00
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 2.7D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      1.000000  1 He py         

 Vector    4  Occ=0.000000D+00  E= 2.524372D+00
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 2.7D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      1.000000  1 He px         

 Vector    5  Occ=0.000000D+00  E= 2.524372D+00
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 2.7D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5      1.000000  1 He pz         


 center of mass
 --------------
 x =   0.00000000 y =   0.00000000 z =   0.00000000

 moments of inertia (a.u.)
 ------------------
           0.000000000000           0.000000000000           0.000000000000
           0.000000000000           0.000000000000           0.000000000000
           0.000000000000           0.000000000000           0.000000000000

  Mulliken analysis of the total density
  --------------------------------------

    Atom       Charge   Shell Charges
 -----------   ------   -------------------------------------------------------
    1 He   2     2.00   1.09  0.91  0.00

       Multipole analysis of the density wrt the origin
       ------------------------------------------------

     L   x y z        total         open         nuclear
     -   - - -        -----         ----         -------
     0   0 0 0      0.000000      0.000000      2.000000

     1   1 0 0      0.000000      0.000000      0.000000
     1   0 1 0      0.000000      0.000000      0.000000
     1   0 0 1      0.000000      0.000000      0.000000

     2   2 0 0     -0.775173      0.000000      0.000000
     2   1 1 0      0.000000      0.000000      0.000000
     2   1 0 1      0.000000      0.000000      0.000000
     2   0 2 0     -0.775173      0.000000      0.000000
     2   0 1 1      0.000000      0.000000      0.000000
     2   0 0 2     -0.775173      0.000000      0.000000


 Parallel integral file used       1 records with       0 large values


 Task  times  cpu:        0.0s     wall:        0.1s
 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls:   40       40      188      120       11        0        0        0     
number of processes/call 1.00e+00 1.00e+00 1.00e+00 0.00e+00 0.00e+00
bytes total:             1.58e+04 9.88e+03 1.34e+03 0.00e+00 0.00e+00 0.00e+00
bytes remote:            0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 2600 bytes
MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        17	        14
	current total bytes		         0	         0
	maximum total bytes		   1052240	  22508936
	maximum total K-bytes		      1053	     22509
	maximum total M-bytes		         2	        23


                                NWChem Input Module
                                -------------------





                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:

                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018

                              AUTHORS & CONTRIBUTORS
                              ----------------------
      E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski, T. P. Straatsma,
     M. Valiev, H. J. J. van Dam, D. Wang, E. Apra, T. L. Windus, J. Hammond,
    J. Autschbach, F. Aquino, J. Mullin, P. Nichols, S. Hirata, M. T. Hackler,
   Y. Zhao, P.-D. Fan, R. J. Harrison, M. Dupuis, D. M. A. Smith, K. Glaesemann,
    J. Nieplocha, V. Tipparaju, M. Krishnan, A. Vazquez-Mayagoitia, L. Jensen,
      M. Swart, Q. Wu, T. Van Voorhis, A. A. Auer, M. Nooijen, L. D. Crosby,
        E. Brown, G. Cisneros, G. I. Fann, H. Fruchtl, J. Garza, K. Hirao,
        R. Kendall, J. A. Nichols, K. Tsemekhman, K. Wolinski, J. Anchell,
       D. Bernholdt, P. Borowski, T. Clark, D. Clerc, H. Dachsel, M. Deegan,
        K. Dyall, D. Elwood, E. Glendening, M. Gutowski, A. Hess, J. Jaffe,
        B. Johnson, J. Ju, R. Kobayashi, R. Kutteh, Z. Lin, R. Littlefield,
    X. Long, B. Meng, T. Nakajima, S. Niu, R. Peverati, L. Pollack, M. Rosing,
          G. Sandrone, M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe,
                                A. Wong, Z. Zhang.

 Total times  cpu:        0.0s     wall:        0.1s
